src/item.cpp

```diff
@@ -4337,6 +4338,11 @@ item::reload_option item::pick_reload_ammo( player &u ) const
         return ( lhs.ammo->ammo_remaining() != 0 ) > ( rhs.ammo->ammo_remaining() != 0 );
     } );
 
+    // NPCs always use the first option
```

Comment:
Arguably a redundant comment as the code below is unambiguous and self documenting